src/test/core.ts

```diff
@@ -842,13 +856,13 @@ describe('snabbdom', function() {
         assert.equal(result.length, 1);
       });
       it('calls `postpatch` after `prepatch` listener', function() {
-        var pre = [], post = [];
-        function preCb(oldVnode, vnode) {
-          pre.push(pre);
+        var pre = 0, post = 0;
+        function preCb() {
+          pre++
         }
-        function postCb(oldVnode, vnode) {
-          assert.equal(pre.length, post.length + 1);
-          post.push(post);
+        function postCb() {
+          assert.equal(pre, post + 1);
+          post++
         }
```

Comment:
I found this one rather unintuitive. It seems to push an array reference into itself. Just for the purpose of counting. So I changed it to counting using numbers to simplify the types involved. Did I miss anything?